src/app/dao/tests/FirebaseFirestoreDAOMock.spec.ts

```diff
@@ -139,34 +160,29 @@ export abstract class FirebaseFirestoreDAOMock<T extends FirebaseJSONObject> imp
             return () => subscription.unsubscribe();
         }
     }
-    private subscribeToMatchers(conditions: [string,
-                                             firebase.firestore.WhereFilterOp,
-                                             unknown][],
+    private subscribeToMatchers(conditions: FirebaseCondition[],
```

Comment:
Dans observingWhere, seul endroit utilisant ce code, il y a "// Note, for now, only check first match field/condition/value at creation, not the added document matching it!"

Est ce toujours d'actualité avec tes updates ?

Reply:
nope, bien vu, enlevé